### PR TITLE
Added Depth First Search in Javascript

### PR DESCRIPTION
**Pull request by pnqphong**
**Fixes [#494](https://github.com/FOSS-UCSC/FOSSALGO/issues/494)**

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

### CODE REVIEW

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    // Create an array / fake "stack" for storing next visiting nodes
+    let stack = [startNode];
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:12]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            graph.edges.filter(edge => edge.start == node && !visited.includes(edge.end)).forEach(item => {
+                stack = [item.end].concat(stack);
+            });
+        }
+    }
+}
+
+function main() {
+    let graph = {
+        nodes: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'],
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:26]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        edges: [
+            { start: 'A', end: 'B' },
+            { start: 'A', end: 'C' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:29]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            { start: 'B', end: 'D' },
+            { start: 'B', end: 'E' },
+            { start: 'D', end: 'E' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:32]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+            { start: 'D', end: 'F' },
+            { start: 'E', end: 'H' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:34]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    }
+}
+
+function main() {
+    let graph = {
+        nodes: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'],
+        edges: [
+            { start: 'A', end: 'B' },
+            { start: 'A', end: 'C' },
+            { start: 'B', end: 'D' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:30]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
+
+const dfs = (graph, startNode) => {
+    // Create an array / fake "stack" for storing next visiting nodes
+    let stack = [startNode];
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        node = stack.shift();
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:9]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+            { start: 'A', end: 'C' },
+            { start: 'B', end: 'D' },
+            { start: 'B', end: 'E' },
+            { start: 'D', end: 'E' },
+            { start: 'D', end: 'F' },
+            { start: 'E', end: 'H' },
+            { start: 'F', end: 'G' }
+        ]
+    }
+    dfs(graph, 'A');
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:38]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
+            console.log(`visited ${node}`);
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:14]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+            
+            // Find direct-connected node and put to stack if it's not visited
+            graph.edges.filter(edge => edge.start == node && !visited.includes(edge.end)).forEach(item => {
+                stack = [item.end].concat(stack);
+            });
+        }
+    }
+}
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:22]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 9 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:14]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+        // Take first node from stack to visit
+        node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
+            console.log(`visited ${node}`);
+            
+            // Find direct-connected node and put to stack if it's not visited
+            graph.edges.filter(edge => edge.start == node && !visited.includes(edge.end)).forEach(item => {
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+            });
+        }
+    }
+}
+
+function main() {
+    let graph = {
+        nodes: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'],
+        edges: [
+            { start: 'A', end: 'B' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:28]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+            { start: 'A', end: 'C' },
+            { start: 'B', end: 'D' },
+            { start: 'B', end: 'E' },
+            { start: 'D', end: 'E' },
+            { start: 'D', end: 'F' },
+            { start: 'E', end: 'H' },
+            { start: 'F', end: 'G' }
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:35]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+function main() {
+    let graph = {
+        nodes: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'],
+        edges: [
+            { start: 'A', end: 'B' },
+            { start: 'A', end: 'C' },
+            { start: 'B', end: 'D' },
+            { start: 'B', end: 'E' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:31]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+    let stack = [startNode];
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:13]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+function main() {
+    let graph = {
+        nodes: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H'],
+        edges: [
+            { start: 'A', end: 'B' },
+            { start: 'A', end: 'C' },
+            { start: 'B', end: 'D' },
+            { start: 'B', end: 'E' },
+            { start: 'D', end: 'E' },
+            { start: 'D', end: 'F' },
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:33]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Function declared in a loop contains unsafe references to variable(s) 'node'.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 14 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:31]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 8 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:38]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 5 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:34]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 3 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:29]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 12 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:28]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Function declared in a loop contains unsafe references to variable(s) 'node'.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 9 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:14]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 15 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:13]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 2 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:26]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 16 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:33]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 11 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 9 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:14]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 6 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:30]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 4 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:32]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 7 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:9]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 1 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:12]: Codacy found an issue: ['node' is not defined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 10 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:22]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 13 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:35]: Codacy found an issue: [Strings must use doublequote.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+        // Take first node from stack to visit
+        let node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
+            console.log(`visited ${node}`);
+            
+            // Find direct-connected node and put to stack if it's not visited
+            graph.edges.filter(edge => edge.start == node && !visited.includes(edge.end)).forEach(item => {
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+                stack = [item.end].concat(stack);
+            });
+        }
+    }
+}
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:22]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+    let visited = [];
+
+    while (stack.length) {
+        // Take first node from stack to visit
+        let node = stack.shift();
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
+            console.log(`visited ${node}`);
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:14]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js (hunk 17 above)
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:17]: Codacy found an issue: [Function declared in a loop contains unsafe references to variable(s) 'node'.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+
+        // Check whether the node is not visited 
+        if (visited.indexOf(node) === -1) {
+            visited.push(node);
+            /* eslint-disable no-console */
+            console.log(`visited ${node}`);
+            /* eslint-enable no-console */
+            
+            // Find direct-connected node and put to stack if it's not visited
+            graph.edges.filter(edge => edge.start === node && !visited.includes(edge.end)).forEach((item) => {
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:19]: Codacy found an issue: [Function declared in a loop contains unsafe references to variable(s) 'stack', 'stack'.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:19]: Codacy found an issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

File: algorithms/gr-dfsrh/javascript/dfs.js [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+            { start: "A", end: "B" },
+            { start: "A", end: "C" },
+            { start: "B", end: "D" },
+            { start: "B", end: "E" },
+            { start: "D", end: "E" },
+            { start: "D", end: "F" },
+            { start: "E", end: "H" },
+            { start: "F", end: "G" }
+        ]
+    }
```
Comment (RavinduSachintha) [inline algorithms/gr-dfsrh/javascript/dfs.js:39]: Codacy found an issue: [Missing semicolon.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6396906)

**Comment by pnqphong** [comment]
Hi @Danushka96 , please help me review the PR.

**Review by Danushka96 — APPROVED** [review]

**Comment by Danushka96** [comment]
@all-contributors please add @pnqphong for code and doc

**Comment by Danushka96** [comment]
Hi @pnqphong 
Thanks for your contribution!

**Pull request merged**